sdk/nodejs/cmd/run/run.ts

```diff
@@ -22,8 +22,10 @@ import * as ini from "ini";
 import * as semver from "semver";
 import { ResourceError, RunError } from "../../errors";
 import * as log from "../../log";
+import * as opentelemetry from "@opentelemetry/api";
```

Comment:
Interesting, curious if this works. The wire protocol that Pulumi will accept is Zipkin V1 traces. Does opentelemetry clients support that, e.g. do you see the traces? I now the protocol does not support full gamut of features like reporting events and metrics.

Reply:
I was able to see the traces when running the Jaegar server, and sending it to the Zipkin endpoint using the v1 API. Not seeing the same right now with the Pulumi Zipkin endpoint.